src/Stripe.Tests.XUnit/payment_intents/creating_and_updating_payment_intents.cs

```diff
@@ -0,0 +1,69 @@
+﻿using FluentAssertions;
+using Xunit;
+
+namespace Stripe.Tests.Xunit
+{
+    public class creating_and_updating_payment_intents : IClassFixture<payment_intents_fixture>
+    {
+        private readonly payment_intents_fixture fixture;
+
+        public creating_and_updating_payment_intents(payment_intents_fixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
+        [Fact]
+        public void created_has_the_right_information()
+        {
+            fixture.PaymentIntent.Should().NotBeNull();
+            fixture.PaymentIntent.Amount.Should().Be(fixture.PaymentIntentCreateOptions.Amount);
+            fixture.PaymentIntent.CaptureMethod.Should().Be(fixture.PaymentIntentCreateOptions.CaptureMethod);
+            fixture.PaymentIntent.Currency.Should().Be(fixture.PaymentIntentCreateOptions.Currency);
+            fixture.PaymentIntent.AllowedSourceTypes.Count.Should().Be(fixture.PaymentIntentCreateOptions.AllowedSourceTypes.Count);
```

Comment:
I could not, for the life of me, figure out how to compare the 2 lists. I tried a lot of options but none worked/compiled for some reasons so I settled for the count for now.